### ISSUE TITLE
github colors

### ISSUE THREAD

**Issue by yeKcim** [flopp/GpxTrackPoster#68]
`create_poster --gpx-dir "gpx/" --track-color '#5599ff' --track-color2 '#ffd732' --output "/tmp/GPXPOSTER.svg" --title 2019-2020 --athlete Bob --type github --language fr_FR --year 2019-2020 --circular-rings; done;`

Colors are not used in github. I use these colors for heatmap and circular. As you can see, there is too much yellow in github:

![GPXPOSTER heatmap+circular 2019-2020_90dpi](https://user-images.githubusercontent.com/10558744/93466607-cc31af80-f8ec-11ea-92e0-dcd2f5a8899d.jpg)

(by the way, thanks a lot for gpxtrackposter ♥)

**Comment by yihong0618** [comment]
@yeKcim 
Yes,  when I  added the type `github` I change the logic, the `github` color is based on distance~

like e.g.
```python
create_poster --type github --gpx-dir "my-tracks" --language "de" --title "Running" \
    --athlete "Florian Pigorsch" --special-distance 10 --special-distance2 10 --special-color yellow --special-color2 red
```
